docs/nodes/config/ui-base.md

```diff
@@ -24,8 +24,8 @@ Some details here about the ui-base config node
 
 <SideBySideImages
     caption="Example of how the 'Collapsing' option looks when open (left) and closed (right)."
-    left="../../public/images/node-examples/ui-base-layout-default-open.png"
-    right="../../public/images/node-examples/ui-base-layout-sidebar-closed.png"
+    left="/images/node-examples/ui-base-layout-default-open.png"
+    right="/images/node-examples/ui-base-layout-sidebar-closed.png"
```

Comment:
Should these be absolute `/.....` urls Joe? 

Will that affect the recent work on making dashboard 2 work behind proxies?

Reply:
these are our online docs changes :)

Reply:
NP